io.openems.edge.pwm.api/src/io/openems/edge/pwm/api/Pwm.java

```diff
@@ -0,0 +1,86 @@
+package io.openems.edge.pwm.api;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.channel.Unit;
+import io.openems.common.types.OpenemsType;
+import io.openems.edge.common.channel.Channel;
+import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.channel.WriteChannel;
+import io.openems.edge.common.component.OpenemsComponent;
+
+public interface Pwm extends OpenemsComponent {
```

Comment:
PWM könnte auch thematisch in das IO-API-Bundle passen, oder?

Reply:
Stimmt, werden wir rüberziehen danke :)